packages/ember-glimmer/lib/setup-registry.js

```diff
@@ -14,15 +14,24 @@ export function setupApplicationRegistry(registry) {
   registry.register('renderer:-dom', InteractiveRenderer);
   registry.register('renderer:-inert', InertRenderer);
 
-  let { DOMChanges, DOMTreeConstruction } = require('ember-glimmer/dom');
+  let { DOMChanges, DOMTreeConstruction, NodeDOMTreeConstruction } = require('ember-glimmer/dom');
+
 
   registry.register('service:-dom-changes', {
     create({ document }) { return new DOMChanges(document); }
   });
 
-  registry.register('service:-dom-tree-construction', {
-    create({ document }) { return new DOMTreeConstruction(document); }
-  });
+  if (typeof document !== 'undefined') {
```

Comment:
It's not the default export for some reason: https://github.com/emberjs/ember.js/blob/master/packages/ember-environment/lib/index.js#L106

So you'll need `import { environment } ...`